docs/serving/services/creating-services.md

```diff
@@ -1,76 +1,71 @@
 ---
 title: "Creating Knative services"
-weight: 10
+linkTitle: "Creating a service"
+weight: 02
 type: "docs"
 ---
 
-Knative Services are used to deploy an application. Each Knative Service is defined by a Route and a Configuration, which have the same name as the Service, contained in a YAML file. Every time the Configuration is updated, a new Revision is created.
+To create an application using Knative, you must create a YAML file that defines a Knative service. This YAML file specifies metadata about the application, points to the hosted image of the app and allows the service to be configured.
```

Comment:
I'm going to `/lgtm` and `/approve`, but I'm not sure about the "you must create a YAML file", since we also support using the `kn` CLI client and may have other mechanisms (web form?) in the future.